### PR TITLE
Changed autocomplete to use either connection or result paths

### PR DESCRIPTION
**Pull request by EduanMullerTG**

### CODE REVIEW

File: src/Autocomplete/index.tsx [hunk 1]
```diff
@@ -100,14 +102,25 @@ const Autocomplete: React.FC<Props> = ({
...
+      if (connectionName) {
+        const edges = _.get(result.data, `${connectionName}.edges`)
+        suggestions = _.map(edges, (edge: any) => ({
+          label: extractLabel(edge.node),
+          value: extractValue(edge.node),
+        }))
+      }
+      else if (resultPath) {
+        const list = _.get(result.data, `${resultPath}`)
+        console.log(list)
```
Comment (bevanmw) [inline src/Autocomplete/index.tsx]: Stray console.log

File: src/Autocomplete/index.tsx [hunk 2]
```diff
@@ -100,14 +102,25 @@ const Autocomplete: React.FC<Props> = ({
...
-        label: extractLabel(edge.node),
-        value: extractValue(edge.node),
-      }))
+      if (connectionName) {
+        const edges = _.get(result.data, `${connectionName}.edges`)
+        suggestions = _.map(edges, (edge: any) => ({
+          label: extractLabel(edge.node),
+          value: extractValue(edge.node),
+        }))
+      }
```
Comment (andreit) [inline src/Autocomplete/index.tsx:111]: Did you run prettier? This looks inconsistent.

File: src/Autocomplete/index.tsx [hunk 3]
```diff
@@ -100,14 +102,25 @@ const Autocomplete: React.FC<Props> = ({
...
+      else if (resultPath) {
+        const list = _.get(result.data, `${resultPath}`)
+        console.log(list)
+        suggestions = _.map(list, (item: any) => ({
+          label: extractLabel(item),
+          value: extractValue(item),
+        }))
+        console.log(suggestions)
```
Comment (andreit) [inline src/Autocomplete/index.tsx:119]: This `console.log` can be removed

File: src/Autocomplete/index.tsx [hunk 4]
```diff
@@ -100,14 +102,25 @@ const Autocomplete: React.FC<Props> = ({
...
+      }
```
Comment (andreit) [inline src/Autocomplete/index.tsx:120]: I think its a good idea to add an `else` that just logs a warning indicating that both `resultPath` and `connectionName` are not set

File: src/Autocomplete/index.tsx (hunk 2 above)
Comment (EduanMullerTG) [reply src/Autocomplete/index.tsx:111]: Doesn't prettier run on commit?

**Review by bevanmw — APPROVED** [review]

**Review by andreit — APPROVED** [review]

**Pull request merged**